src/Augmentation/ui/PurchasableAugmentations.tsx

```diff
@@ -0,0 +1,253 @@
+          <>
+            <Report fontSize="small" sx={{ mr: 1 }} />
+            Missing {props.aug.prereqs.length - ownedPreReqs.length} pre-requisite(s)
+          </>
+        )}
+      </Typography>
+    </Tooltip>
+  );
+};
+
+interface IExclusiveProps {
+  player: IPlayer;
+  aug: Augmentation;
+}
+
+const Exclusive = (props: IExclusiveProps): React.ReactElement => {
+  return (
+    <Tooltip
+      title={
+        <Typography sx={{ color: Settings.theme.money }}>
+          This Augmentation can only be acquired from the following source(s):
+          <ul>
+            <li>
+              <b>{props.aug.factions[0]}</b> faction
+            </li>
+            {props.player.canAccessGang() && !props.aug.isSpecial && (
+              <li>
+                Certain <b>gangs</b>
+              </li>
+            )}
+            {props.player.canAccessGrafting() &&
+              !props.aug.isSpecial &&
+              props.aug.name !== AugmentationNames.TheRedPill && (
+                <li>
+                  <b>Grafting</b>
+                </li>
+              )}
+          </ul>
+        </Typography>
+      }
+    >
+      <NewReleases sx={{ ml: 1, color: Settings.theme.money, transform: "rotate(180deg)" }} />
+    </Tooltip>
+  );
+};
+
+interface IReqProps {
+  value: string;
+  color: string;
+  fulfilled: boolean;
+}
+
+const Requirement = (props: IReqProps): React.ReactElement => {
+  return (
+    <Typography sx={{ display: "flex", alignItems: "center", color: props.color }}>
+      {props.fulfilled ? <CheckBox sx={{ mr: 1 }} /> : <CheckBoxOutlineBlank sx={{ mr: 1 }} />}
+      {props.value}
+    </Typography>
+  );
+};
+
+interface IPurchasableAugsProps {
+  augNames: string[];
+  ownedAugNames: string[];
+  player: IPlayer;
+
+  canPurchase: (player: IPlayer, aug: Augmentation) => boolean;
+  purchaseAugmentation: (player: IPlayer, aug: Augmentation, showModal: (open: boolean) => void) => void;
+
+  rep?: number;
+  sleeveAugs?: boolean;
+  faction?: Faction;
+}
+
+export const PurchasableAugmentations = (props: IPurchasableAugsProps): React.ReactElement => {
+  return (
+    <Container
+      maxWidth="md"
+      disableGutters
+      sx={{ mx: 0, display: "grid", gridTemplateColumns: "repeat(1, 1fr)", gap: 1 }}
+    >
+      {props.augNames.map((augName: string) => (
+        <PurchasableAugmentation key={augName} parent={props} augName={augName} owned={false} />
+      ))}
+      {props.ownedAugNames.map((augName: string) => (
+        <PurchasableAugmentation key={augName} parent={props} augName={augName} owned={true} />
+      ))}
+    </Container>
+  );
+};
+
+interface IPurchasableAugProps {
+  parent: IPurchasableAugsProps;
+  augName: string;
+  owned: boolean;
+}
+
+export function PurchasableAugmentation(props: IPurchasableAugProps): React.ReactElement {
+  const [open, setOpen] = useState(false);
+
+  const aug = Augmentations[props.augName];
+
+  const cost = props.parent.sleeveAugs ? aug.startingCost : aug.baseCost;
```

Comment:
just a note, incase mine makes it through first starting cost is no more and base cost will be starting cost, and a function on the augmentation is being added to get the current cost instead

Reply:
sounds good, thanks for the heads up!